bug-reports.md

```diff
@@ -12,11 +12,11 @@ Once you've done a bit of searching and discovered there isn't prior art for you
 
 **Summary**
 
-In narrative form, such as “While testing feature x, I encountered z.”
+In narrative forms, such as “While testing feature x, I encountered z.”
```

Comment:
This is not about forms, but about a form of communication. Please revert this change.